docs/services/topology-contacts-data.md

```diff
@@ -70,6 +70,51 @@ If you remove the last resource in the resource group yaml file, you should remo
 downtime yaml files as well.
 
 
+### Reviewing project PRs
+
+New projects are typically created by the Research Facilitation team.
+Here are a few things to check:
+
+-   Does the "Name" in the YAML match the file name of the project?
+
+-   Did osg-bot warn about a "New Organization"?
+    If so, search around in the projects directory and make sure the "Organization" in the YAML
+    is not a typo or alternate spelling for an existing organization.
```

Comment:
> Search around in the projects directory...

Doesn't `osg-bot` warn about the specific problematic YAML? I think we can provide better guidance than looking at all the project files. 

> is not a typo or alternate spelling for an existing organization

How does someone know if the organization already exists?

Reply:
CI tests for projects are moderately nonexistent. The only thing that gets checked is whether the webapp is able to load all the data and render it as XML according to the schema -- which means basically all required fields are there and there are no syntax errors.

To find out if an organization exists, grep around in the projects directory for alternate spellings.